common/middleware/ensure-authenticated.test.js

```diff
@@ -7,13 +7,16 @@ describe('Authentication middleware', function() {
     let req, res, nextSpy
 
     beforeEach(function() {
+      const headerStub = sinon.stub()
+      headerStub.returns('')
       nextSpy = sinon.spy()
       req = {
         url: '/url',
         originalUrl: '/test',
         session: {
           authExpiry: null,
         },
+        header: headerStub,
```

Comment:
I find it neater to write this as:

```javscript
...
header: sinon.stub().returns(''),
...
```

Then testing assertions like this:

```javascript
expect(req.header).to.be.calledOnceWithExactly(...)
```

Reply:
👍